arch/arm/core/aarch64/reset.S

```diff
@@ -115,8 +115,31 @@ SECTION_SUBSEC_FUNC(TEXT,_reset_section,__start)
 	eret
 
 2:
-	/* Booting from EL2 is not supported */
-	b	.
+	mov_imm	x1, SCTLR_EL2_RES1
```

Comment:
is this still related to this commit? It seems to me that you are entering in EL1 so this code is not used?

Reply:
This piece code is to let me could boot Zephyr using EL2 U-Boot on i.MX8MM. Using Qemu test, this piece code is not needed.

Reply:
I see. I embedded part of that into #32394

Reply:
ok, I'll drop this change in my PR.

Reply:
Wait for that. Please, do that only after that PR is merged.